### PR TITLE
0x0c Constructor Drone

### PR DESCRIPTION
**Pull request by dfuchs2**

### CODE REVIEW

File: CharacterCreation/01_02_Species_ECSC_Security_Drone.txt [hunk 1]
```diff
@@ -214,3 +214,18 @@ in space and in zero gravity. Cryo Drones get a +20 to Reflex saves to move in
 zero gravity and +20 to Acrobatics checks to maneuver in zero gravity as well. 
 They have a -20 to skill checks involving Swim. Cryo Drones with this trait only 
 reduce plasma burn tokens every 3 turns instead of every other turn. 
+
+===== Series 0x0c "Constructor" =====
+
+    During the mid 2400s, the ECSC had many varieties of purpose-built construction 
+drones in service. The Series Twelve Constructor Drone was the response by the ECSC 
+to replace them all with an all-purpose unit that could be easily utilized on most 
+inhabited worlds. The Constructor Drone is now a common sight throughout ECSC space
```
Comment (bleehu) [inline CharacterCreation/01_02_Species_ECSC_Security_Drone.txt:223]: This is _Awesome_! I also love how 12 in hex is "c" for construction!

File: CharacterCreation/01_02_Species_ECSC_Security_Drone.txt [hunk 2]
```diff
@@ -214,3 +214,18 @@ in space and in zero gravity. Cryo Drones get a +20 to Reflex saves to move in
...
+===== Series 0x0c "Constructor" =====
+
+    During the mid 2400s, the ECSC had many varieties of purpose-built construction 
+drones in service. The Series Twelve Constructor Drone was the response by the ECSC 
+to replace them all with an all-purpose unit that could be easily utilized on most 
+inhabited worlds. The Constructor Drone is now a common sight throughout ECSC space 
+where there is work to be done. Series Twelve Drones are usually black, gray, or 
+white, though it is common for military units to be repainted to fit with the 
+uniforms of everyone else. Constructor Drones also come with an optional high 
+visiblity yellow pattern package. The Series Twelve has a bulkier frame than other
```
Comment (bleehu) [inline CharacterCreation/01_02_Species_ECSC_Security_Drone.txt:227]: Rev says "GOOD!" I love it too!

**Review by bleehu — APPROVED** [review]
This is Jexcellent! Shipit!

**Review by Turtlelord26 — COMMENTED** [review]
This is an excellent concept, and 🐢 approves.

However, the flavor text for Security Drones implies pretty directly that all Security Drones are military or security hardware. This is in opposition to your general-purpose construction flavor. I would like to see these two flavors brought into more harmony, possibly by one of the ways below
* Alter Security Drone flavor to allow inclusion of construction models.
* Alter Construction drone flavor to more specifically be combat engineers.
* Save the Construction Drone subspecies for when @Bleehu reintroduces the Multipurpose Drone race.

One other question: How do we flavor-justify a Series 12 Constructor Security Drone as a class other than Engineer? How might such a thing happen? If we have answers we should probably sneak them into the prose so that our players don't necessarily have to work so hard to find them.

**Review by Reveraine — APPROVED** [review]
Excellent

**Pull request merged**